concepts/05 Widgets/Tabs/15 Control the Behavior.md

```diff
@@ -1,64 +1,420 @@
 An end user can select **Tabs** items in two different modes: *'single'* (by default) or *'multiple'*. You can use the [selectionMode](/api-reference/10%20UI%20Widgets/dxTabs/1%20Configuration/selectionMode.md '/Documentation/ApiReference/UI_Widgets/dxTabs/Configuration/#selectionMode') option to change the mode. 
 
+---
+##### jQuery
+
+    <!--HTML-->
+    <div id="tabsContainer"></div>
+
     <!--JavaScript-->
     $(function() {
         $("#tabsContainer").dxTabs({
-            items: tabItems,
+            dataSource: [
```

Comment:
The `dataSource` isn't needed as well because it isn't relevant to what we show. Please revert.